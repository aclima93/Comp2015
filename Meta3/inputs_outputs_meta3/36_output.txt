Line 12, col 25: Incompatible type in repeat-until statement (got _integer_, expected _boolean_)
