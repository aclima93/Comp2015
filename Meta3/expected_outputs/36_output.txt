Line 12, col 21: Incompatible type in repeat-until statement (got _integer_, expected _boolean_)
